docs/ops/sort/MulticlassNMS_9.md

```diff
@@ -0,0 +1,175 @@
+  * **Range of values**: an integer
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *no*
+
+* *background_class*
+
+  * **Description**: the background class id.
+  * **Range of values**: an integer
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all classes.
+  * **Required**: *no*
+
+* *normalized*
+
+  * **Description**: *normalized* is a flag that indicates whether `boxes` are normalized or not.
+  * **Range of values**: true or false
+    * *true* - the box coordinates are normalized.
+    * *false* - the box coordinates are not normalized.
+  * **Type**: boolean
+  * **Default value**: True
+  * **Required**: *no*
+
+* *nms_eta*
+
+  * **Description**: eta parameter for adaptive NMS.
+  * **Range of values**: a floating-point number in close range `[0, 1.0]`.
+  * **Type**: `float`
+  * **Default value**: `1.0`
+  * **Required**: *no*
+
+**Inputs**:
+
+There are 2 kinds of input formats. The first one is -
+*   **1**: `boxes` - tensor of type *T* and shape `[num_batches, num_boxes, 4]` with box coordinates. The box coordinates are layout as `[xmin, ymin, xmax, ymax]`. **Required.**
+
+*   **2**: `scores` - tensor of type *T* and shape `[num_batches, num_classes, num_boxes]` with box scores. The tensor type should be same with `boxes`. **Required.**
+
+The second format is -
+*   **1**: `boxes` - tensor of type *T* and shape `[num_classes, num_boxes, 4]` with box coordinates. The box coordinates are layout as `[xmin, ymin, xmax, ymax]`. **Required.**
+
+*   **2**: `scores` - tensor of type *T* and shape `[num_classes, num_boxes]` with box scores. The tensor type should be same with `boxes`. **Required.**
+
+*   **3**: `roisnum` - tensor of type *T_IND* and shape `[num_batches]` with box numbers of each image, which sum up to `num_boxes`. **Required.**
+
+**Outputs**:
+
+*   **1**: `selected_outputs` - tensor of type *T* which should be same with `boxes` and shape `[number of selected boxes, 6]` containing the selected boxes with score and class as tuples `[class_id, box_score, xmin, ymin, xmax, ymax]`.
+
+*   **2**: `selected_indices` - tensor of type *T_IND* and shape `[number of selected boxes, 1]` the selected indices in the flattened `boxes`, which are absolute values cross batches. Therefore possible valid values are in the range `[0, num_batches * num_boxes - 1]`.
+
+*   **3**: `selected_num` - 1D tensor of type *T_IND* and shape `[num_batches]` representing the number of selected boxes for each batch element.
+
+When there is no box selected, `selected_num` is filled with `0`. `selected_outputs` is an empty tensor of shape `[0, 6]`, and `selected_indices` is an empty tensor of shape `[0, 1]`.
+
+**Types**
+
+* *T*: floating-point type.
+
+* *T_IND*: `int64` or `int32`.
+
+**Example**
+
+```xml
+<layer ... type="MulticlassNonMaxSuppression" ... >
+    <data sort_result="score" output_type="i64"/>
```

Comment:
where are all the attributes?

Reply:
they could be optional as they have defaults. Need list them all here?

Reply:
and they are optional. anyway, I added all them in examples.